README.md

```diff
@@ -4,7 +4,48 @@ This is still experimental. Your code should be safe,
 since IPython handles saving and loading notebooks in another process, but
 you'll lose all your variables if it crashes.
 
-## Installing
+## Installing 
+   
+Requirements:
+
+* [IPython](http://ipython.org/), version 3.1 or later. If you already have a 
+  Python environment set up, install IPython using your preferred tools. If 
+  not, installing [Anaconda](http://continuum.io/downloads) is the quickest 
+  way to get everything you need. IPython 3.0 does not work due to a bug in 
+  that version. Earlier versions of IPython are now unsupported.
+* A current [R installation](http://www.r-project.org/).
+  
+We provide source and windows binary packages of the needed package. They can be installed 
+with the following lines in an R console: 
+
+```r
+install.packages(c("rzmq","repr","IRkernel","IRdisplay"), repos="http://irkernel.github.io/")
+IRkernel::installspec()
+```
+
+On Mac, you probably need to `install.packages(... , type="source")`.
```

Comment:
Anyway: I think the most sensible thing would be to have someone compile the mac packages and install them via drat in that repo...